### PR TITLE
[Static reference] LPS-170588 Remove method reference by using snapshot in module roles

### PR DESCRIPTION
**Pull request by W2jx**

### CODE REVIEW

**Comment by zhangyan0701** [comment]
ci:test:roles

**Comment by W2jx** [comment]
no conflicts(4/7 16:38)

**Comment by W2jx** [comment]
no conflicts(4/10 8:33)

**Comment by W2jx** [comment]
no conflicts(4/11 9:50)

File: modules/apps/roles/roles-admin-web/src/main/java/com/liferay/roles/admin/web/internal/display/context/SegmentsEntryDisplayContext.java [hunk 1]
```diff
@@ -34,20 +35,18 @@
...
 import javax.servlet.http.HttpServletRequest;
 
-import org.osgi.service.component.annotations.Component;
-import org.osgi.service.component.annotations.Reference;
-
 /**
  * @author Pei-Jung Lan
  */
-@Component(service = {})
 public class SegmentsEntryDisplayContext {
```
Comment (dantewang) [inline modules/apps/roles/roles-admin-web/src/main/java/com/liferay/roles/admin/web/internal/display/context/SegmentsEntryDisplayContext.java:41]: @W2jx This class is a util, please rename it and move it to proper package.

**Review by dantewang — CHANGES_REQUESTED** [review]

**Comment by dantewang** [comment]
ci:forward

cc @shuyangzhou @tinatian

**Comment by dantewang** [comment]
@W2jx Sorry typed wrong content. Please ignore the forward and make changes as commented.

**Pull request closed without merging**